docs/xms-pageable-for-get-list-calls.md

```diff
@@ -0,0 +1,143 @@
+# XmsPageableForGetListCalls
+
+## Category
+
+ARM Error
+
+## Applies to
+
+ARM OpenAPI(swagger) specs
+
+## Related ARM Guideline Code
+
+- RPC-Get-V1-13
+
+## Output Message
+
+For all LIST APIs (a.k.a collection GETs), it is important to include the `x-ms-pageable` property.
+
+## Description
+
+`x-ms-pageable` indicates that the results of a GET call may be paginated. All LIST APIs (a.k.a collection GETs) should include this annotation.
+
+## How to fix the violation
+
+Please include `x-ms-pageable` property to the collection GET calls.
```

Comment:
>property [](http://example.com/codeflow?start=31&length=8)

Term "property" is ambiguous, I mistook it be defined within properties.
We need to consider using a different term?

Reply:
extension is a good term to use